src/test/java/org/itsallcode/openfasttrace/importer/TestSpecificationListBuilder.java

```diff
@@ -22,37 +24,150 @@
  * #L%
  */
 
-
 import static org.hamcrest.Matchers.equalTo;
 import static org.junit.Assert.assertThat;
 
+import java.util.Arrays;
+import java.util.List;
+
+import org.itsallcode.openfasttrace.core.ItemStatus;
+import org.itsallcode.openfasttrace.core.SpecificationItem;
 import org.itsallcode.openfasttrace.core.SpecificationItemId;
-import org.itsallcode.openfasttrace.importer.SpecificationListBuilder;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.Test;;
 
 public class TestSpecificationListBuilder
 {
-    private static final SpecificationItemId ITEM_ID = SpecificationItemId.parseId("foo~bar~1");
-    private SpecificationListBuilder builder;
 
-    @Before
-    public void setUp()
+    private final static SpecificationItemId ID = SpecificationItemId.parseId("feat~id~1");
+
+    @Test
+    public void testBuildBasicItem()
     {
-        this.builder = new SpecificationListBuilder();
+        final String expectedTitle = "title";
+        final String expectedDescription = "description";
+        final SpecificationListBuilder itemsBuilder = createBasicListBuilder();
+        itemsBuilder.appendDescription(expectedDescription);
+        itemsBuilder.setTitle(expectedTitle);
+        final List<SpecificationItem> items = itemsBuilder.build();
+        assertThat(items.size(), equalTo(1));
+        assertThat(items.get(0).getId(), equalTo(ID));
+        assertThat(items.get(0).getDescription(), equalTo("description"));
+        assertThat(items.get(0).getTitle(), equalTo("title"));
```

Comment:
Use variables `expectedTitle` / `expectedDescription` defined earlier, maybe extract variables into constants so that they can be used in other tests.